### ISSUE TITLE
CTD Approaching Ancient's Ascent from Bonechill Passage with ParallaxGen v0.7.0/0.7.1 Output but not v0.6.0 Output

### ISSUE THREAD

**Issue by skyrim50se** [hakasapl/ParallaxGen#107]
I have attached .NETScriptFramework error logs as well as the ParallaxGen v0.7.1 log and ParallaxGen v0.6.0 log. Let me know if there is any additional information I can get you to help sort this out. I believe I checked the equivalent settings on v0.7.0/0.7.1 compared to v0.6.0.
[Crash_2024_11_19_21-45-1.txt](https://github.com/user-attachments/files/17826449/Crash_2024_11_19_21-45-1.txt)
[Crash_2024_11_19_21-26-15.txt](https://github.com/user-attachments/files/17826450/Crash_2024_11_19_21-26-15.txt)
[Crash_2024_11_19_21-18-20.txt](https://github.com/user-attachments/files/17826451/Crash_2024_11_19_21-18-20.txt)
[ParallaxGen-v6.log](https://github.com/user-attachments/files/17826454/ParallaxGen-v6.log)
[ParallaxGen-v7.0.1.log](https://github.com/user-attachments/files/17826456/ParallaxGen-v7.0.1.log)

**Comment by hakasapl** [comment]
@skyrim50se thanks for your detailed logs. Could you also send us these meshes from your load order and from the PG output so we can compare what PG did?

* `data\MESHES\dungeons\nordic\chambers\powerwordtah.nif`
* `data\MESHES\dungeons\nordic\chambers\powerwordraan.nif`

**Comment by skyrim50se** [comment]
Thanks for tracking down the NIFs involved. I am using the Word Wall
Transparency Fix for ENB which provides these NIFs in my LO. These are
absent from my ParallaxGen 0.6.0 output, but present in the 0.7.1 output. I
attached the complete set from Word Wall Transparency and ParallaxGen 0.7.1
output. I needed to generate the output again from 0.7.1 as I didn't save
it from before.

**Comment by hakasapl** [comment]
I don't see the NIFs attached - maybe attaching via email doesn't work?

**Comment by skyrim50se** [comment]
Looks like GitHub doesn't accept 7zip files. Here ya go!
[PGen0.7.1 Chambers.zip](https://github.com/user-attachments/files/17853330/PGen0.7.1.Chambers.zip)
[WWTF Chambers Folder.zip](https://github.com/user-attachments/files/17853331/WWTF.Chambers.Folder.zip)

**Comment by hakasapl** [comment]
Apologies for the delay, just coming back to this. What mod does `textures\dungeons\dragonrunealphabet_p.dds` come from? It patched with complex material with that file being a CM map (or at least PG thinks it is)

**Comment by skyrim50se** [comment]
Rustic Reliefs - Parallax v1.5 - https://www.nexusmods.com/skyrimspecialedition/mods/34684

[dragonrunealphabet_p.zip](https://github.com/user-attachments/files/18017043/dragonrunealphabet_p.zip)

**Comment by hakasapl** [comment]
I think PG thinks that parallax map is a CM map because it is used as an env mask in a mesh that PG scanned. Could you run with the `-vv` CLI argument and zip up your logs folder and send it? The extended logs should have this info.

**Comment by skyrim50se** [comment]
I added -vv to the Arguments field in MO2. Here are the logs.
[log0-99.zip](https://github.com/user-attachments/files/18017806/log0-99.zip)
[log100-197.zip](https://github.com/user-attachments/files/18017808/log100-197.zip)

**Comment by hakasapl** [comment]
That's really strange. The mod that won is correct when patching:
```
[2024-12-04 21:21:17.638] [trace] [meshes\dungeons\nordic\chambers\powerwordraan.nif] [330 / PowerWordRaanMesh] Winning mod: RUSTIC RELIEFS - Parallax
```

and PG thinks this is a height map:

```
[2024-12-04 20:51:15.761] [trace] ModManagerDirectory | Adding Files to Map : textures\dungeons\dragonrunealphabet_p.dds -> RUSTIC RELIEFS - Parallax
[2024-12-04 20:53:33.711] [trace] Mapping Textures | Mapping Result | Texture: textures\dungeons\dragonrunealphabet_p.dds | Slot: 3 | Type: height
```

From the output that was just generated by the logs you send the mesh `data\MESHES\dungeons\nordic\chambers\powerwordraan.nif`? Based on this log and the mesh you sent before they don't make sense, the mesh you sent before was patched for CM but this log says it patched it for parallax. I guess it could be some logging bug with PG but if you have the mesh handy that would be good to check.

**Comment by skyrim50se** [comment]
[powerwordraan.zip](https://github.com/user-attachments/files/18018238/powerwordraan.zip)

**Comment by skyrim50se** [comment]
Oh, and I thought I had the setting on for parallax textures to be converted into CM textures because I use BDS3.

**Comment by hakasapl** [comment]
I think I found the issue, will fix in 0.7.2.
